app/forms/nominate_induction_tutor_form.rb

```diff
@@ -5,70 +5,94 @@ class NominateInductionTutorForm
-  validate :email_is_not_in_use, on: :email
-  validate :name_matches, on: :email
+  # Validations
+  validates :email, presence: true, notify_email: true, on: %i[email]
+  validates :full_name, presence: true, on: %i[full_name email]
+  validates :school, presence: true, on: %i[email]
+  validate :not_current_sit, on: %i[email]
+  validate :name_matches, on: %i[email]
 
+  # Instance Methods
   def attributes
-    {
-      full_name:,
-      email:,
-      token:,
-    }
+    { email:, full_name:, token: }
   end
 
-  def school
-    if school_id
-      School.find school_id
-    else
-      NominationEmail.find_by(token:).school
+  def save!
+    raise errors.full_messages unless valid?(:email)
+
+    ActiveRecord::Base.transaction do
+      remove_current_sits
+      ensure_user_persisted
+      add_school_to_sit
+      inform_user
     end
+
+    true
   end
 
-  def existing_name
-    user_scope = user_id.present? ? User.induction_coordinators.where.not(id: user_id) : User.induction_coordinators
-    existing_user = user_scope.find_by(email:)
-    return if existing_user.blank?
+private
 
-    existing_user.full_name
+  def add_school_to_sit
+    sit_profile.schools << school
   end
 
-  def name_different?
-    existing_name.present? && existing_name != full_name
+  def different_name?
+    user.full_name != full_name
   end
 
-  def email_already_taken?
-    Identity.find_user_by(email:)
+  def ensure_user_persisted
+    user.save! unless user.persisted?
   end
 
-private
+  def inform_user
+    SchoolMailer.with(sit_profile:, school:, start_url:, step_by_step_url:)
+                .nomination_confirmation_email.deliver_later
+  end
 
-  def email_is_not_in_use
-    if email_already_taken?
-      other_school = school_using_this_email(email)
-      if other_school.present?
-        errors.add(:email, "The email address #{email} is already used by a user at #{other_school.name}")
-      else
-        errors.add(:email, "The email address #{email} is already in use")
-      end
+  def name_matches
+    if different_name?
+      errors.add(:full_name, "A user with a different name (#{user.full_name}) has already been registered with this email address. Please, change the name or email you have entered.")
     end
   end
 
-  def different_name
-    if name_different?
-      errors.add(:full_name, "An induction tutor has already been nominated using this email address, with the name #{existing_name}. The name you entered is #{full_name}.")
+  def not_current_sit
+    errors.add(:email, "The user with email #{user.email} is already an induction coordinator at #{school.name}") if sit_at_school?
+  end
+
+  def remove_current_sits
+    school.induction_coordinators.each do |sit|
+      next sit.induction_coordinator_profile.schools.delete(school) if sit.schools.count > 1
+      next sit.induction_coordinator_profile.destroy! if sit.teacher_profile.present? || sit.npq_registered?
+
+      sit.destroy!
     end
```

Comment:
Should we be removing all SITs in this instance?  Would that be the desired outcome of nominating a new SIT?

Reply:
Why only the first one?

Reply:
Exactly. I just wanted to ensure we knew why you were removing all, as I don't think we have much option here at the moment.